ModularTegustation/_adventure_console/adventure_events/abnormality/he/tsa_woods.dm

```diff
@@ -0,0 +1,66 @@
+/datum/adventure_event/tsa
+	name = "Oddity in a Forest"
+	desc = "YOU FIND A BUILDING"
+//	require_abno = /mob/living/simple_animal/hostile/abnormality/tsa_woods
+	adventure_cords = list(
+		"Get in line.<br>\
+		Don't cut!<br>\
+		A massive line has formed in an outdoor wooden corridor before you.<br>\
+		It would take an incredibly long time to reach the end, and you've yet to join it.<br>\
+		However, a forest completely surrounds the area. What are they getting in line for?",
+
+		"You enter the line. <br>\
+		The path snakes around a central building, and it takes roughly \
+		an hour to reach the end of the line. <br> \
+		At the end, you see people tearing their own skins and placing them into bins.",
+
+		"You conform to the rules, as everyone else has.<br> \
+		No matter how painful or unusual it may be.<br> \
+		After the traumatic ordeal, you are instructed to wear your skin as if it were any article of clothing.<br> \
+		However, the mental scars remain.",
+
+		"The building is so crowded, you can't escape by conventional means. \
+		You need to think of a way to get out of here.",
+
+		"If this place operates the way you think it does, then... <br> \
+		You scream 'BOMB!' as loud as you can. <br> \
```

Comment:
The rest of the story is good but this part might make people give it some weird looks. But that might just be me being weird.